datasafe-encryption/datasafe-encryption-impl/src/main/java/de/adorsys/datasafe/encrypiton/impl/keystore/KeyStoreServiceImpl.java

```diff
@@ -83,6 +83,26 @@ public KeyStore createKeyStore(KeyStoreAuth keyStoreAuth,
         return result;
     }
 
+    @Override
+    @SneakyThrows
+    public List<PublicKeyIDWithX509Cert> getCertificates(KeyStoreAccess keyStoreAccess) {
```

Comment:
It must be getPublicKeys function, don't duplicate it

Reply:
done